Workbooks/Azure Stack HCI/ServerUsage/ServerUsage.workbook

```diff
@@ -0,0 +1,365 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "2c3bd553-1307-4363-97c7-e2b1bc3d543e",
+            "version": "KqlParameterItem/1.0",
+            "name": "timeRange",
+            "label": "Time range",
+            "type": 4,
+            "isRequired": true,
+            "value": {
+              "durationMs": 86400000
+            },
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                },
+                {
+                  "durationMs": 1209600000
+                },
+                {
+                  "durationMs": 2419200000
+                },
+                {
+                  "durationMs": 2592000000
+                }
+              ]
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "timeContextFromParameter": "timeRange"
+          },
+          {
+            "id": "54788f5d-0e4c-4dfb-937b-6e80af8572fa",
+            "version": "KqlParameterItem/1.0",
+            "name": "resourceId",
+            "type": 1,
+            "isRequired": true,
+            "isHiddenWhenLocked": true,
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "timeContextFromParameter": "timeRange"
+          },
+          {
+            "id": "9b253dd6-24fa-4c72-bc24-0018d2e12ad2",
+            "version": "KqlParameterItem/1.0",
+            "name": "resource",
+            "type": 5,
+            "isRequired": true,
+            "query": "where type =~ 'microsoft.azurestackhci/clusters'\r\n| where id =~ '{resourceId}'",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.azurestackhci/clusters": true
+              },
+              "nameFilter": "'n26-20-01-c3'",

```

Comment:
is this part needed?

Reply:
This workbook will be opened from link in another workbook. So, the parameter is passed from the workbook and this parameter converting ARM ID from text to resource type. It makes easier to filter query by subscription ID in ARM ID. I'm testing and haven't find any better way to extract subscription ID and filter following queries with it yet. I'll remove or replace this if I find better solution. Please let me know if there is.